packages/react/src/useEntityQuery.ts

```diff
@@ -7,19 +7,29 @@ import { distinctUntilChanged, map } from "rxjs";
 // This does a little more rendering than is necessary when arguments change,
 // but at least it's giving correct results now. Will optimize later!
 
-export function useEntityQuery(fragments: QueryFragment[]) {
+/**
+ * Returns all matching `EntityIndex`es for a given entity query,
+ * and triggers a re-render as new query results come in.
+ *
+ * @param fragments Query fragments to match against, executed from left to right.
+ * @param recomputeOnValueChange False - re-renders only on entity array changes. True - also on component value changes.
+ * @returns Set of entities matching the query fragments.
+ */
+export function useEntityQuery(fragments: QueryFragment[], recomputeOnValueChange = false) {
```

Comment:
Can we make this second argument an object so we can add more options in the future?